aa-sdk/core/src/errors/client.ts

```diff
@@ -88,3 +88,19 @@ export class InvalidNonceKeyError extends BaseError {
     );
   }
 }
+
+/**
+ * Error class denoting that the provided entity id is invalid because it's overriding the native entity id.
```

Comment:
I assume native entityId is the fallback validation entityId right?

Reply:
yep!